Lib/test/test_types.py

```diff
@@ -1360,6 +1360,20 @@ class C: pass
         D = types.new_class('D', (A(), C, B()), {})
         self.assertEqual(D.__bases__, (A1, A2, A3, C, B1, B2))
 
+    def test_get_orig_bases(self):
+        T = typing.TypeVar('T')
+        class A: pass
+        class B(object): pass
+        class C(typing.Generic[T]): pass
+        class D(C[int]): pass
+        class E(C[str], float): pass
```

Comment:
Please also add tests for how the function works on classes inheriting from `types.GenericAlias` instances, e.g.

```py
class F(list[T]): ...
class G(list[int]): ...
```